### ISSUE TITLE
Display List Item: On accessing the list item as highlighted, UI information is conveyed twice. Also, information about all the list item is conveyed, on accessing any of the list item.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3834]
Description: On accessing the list item as highlighted, UI information is conveyed twice. Also, information about all the list item is conveyed, on accessing any of the list item.

Expected: Information should be clear & precisely conveyed by the screen reader, on accessing the UI element.
Multiple speech output should be avoided.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98774795-bf739880-2411-11eb-9c32-5b8f378606d9.png)
